docs/about/team/the-software-development-team.md

```diff
@@ -4,6 +4,24 @@ sidebar_position: 2
 
 # The Software development team
 
+### [Maxence Guindon](https://www.linkedin.com/in/maxenceguindon)
+
+Hello! My name is Maxence Guindon, and I am a backend developer at the Canadian Food
+Inspection Agency AI-Lab.
+
+I’m currently working on Nachet, a web application using computer vision AI models, which is designed to help CFIA seed analysts identify seeds from images taken from digital microscopes. More specifically, my role in
```

Comment:
still not wrapped here

Reply:
Done. Might have unwrapped it accidentaly when I corrected mistake